roofit/roofit/src/RooGaussian.cxx

```diff
@@ -33,14 +33,14 @@ ClassImp(RooGaussian);
 ////////////////////////////////////////////////////////////////////////////////
 
 RooGaussian::RooGaussian(const char *name, const char *title,
-          RooAbsReal& _x, RooAbsReal& _mean,
-          RooAbsReal& _sigma) :
+          RooAbsReal::Ref _x, RooAbsReal::Ref _mean,
```

Comment:
Here I think you don't want that x is a number !

Reply:
I thought why not: it's more consistent like that, and it can also happen that the observable is a number when you try out models with global observables